lib/swagger-ui.js

```diff
@@ -27,25 +27,48 @@ function processRequest(app, req, res) {
     return fs.readFileAsync(filePath)
     .then((body) => {
         if (reqPath === '/index.html') {
+            const css = `
+                /* Removes Swagger's image from the header bar */
+                .topbar-wrapper .link img {
+                    display: none;
+                }
+                /* Adds the application's name in the header bar */
+                .topbar-wrapper .link::after {
+                    content: "${app.info.name}";
+                }
+                /* Removes input field and explore button from header bar */
+                .swagger-ui .topbar .download-url-wrapper {
+                    display: none;
+                }
+                /* Modifies the font in the information area */
+                .swagger-ui .info li, .swagger-ui .info p, .swagger-ui .info table, .swagger-ui .info a {
+                    font-size: 16px;
+                    line-height: 1.4em;
+                }
+                /* Removes authorize button and section */
+                .scheme-container {
+                    display: none
+                }
+            `;
```

Comment:
Some of this functionality seems like it could be a good candidate for a common npm library instead of duplicating the same thing throughout ur various Node services.

Reply:
you are reading my mind @berndsi 

@mateusbs17 would you mind filing a ticket to abstract this out?